crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
IMHO it's still not too bad for the short time we need this change as this anyway gets executed only on the admin node within our development environment used by 3 people.